src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -342,6 +343,12 @@ export default class DissolutionFirm extends Mixins(DateMixin) {
     return new Date(this.getBusinessFoundingDate)
   }
 
+  /** Dissolution Error */
+  private dissolutionError (): String {
+    return this.isTypeFirm && this.getValidateSteps &&
+    !this.getDissolutionDate && 'Dissolution date is required'
+  }
```

Comment:
1. Return type should be `string` (native type) not `String` (object).
2. What does this return if any of the conditions is False? Is `errorMsg` still OK with it?

Reply:
OK, that works but it's complicated to understand/review. How about a simple if-then-else instead?

Reply:
Your fix will probably fix the lint errors, too :)

Reply:
Nice!